tests/Unit/DataStructures/TensorEagerMath/Test_Determinant.cpp

```diff
@@ -22,8 +22,8 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.EagerMath.Determinant",
     {
       tnsr::ij<double, 2, Frame::Grid> matrix{};
       matrix.get<0, 0>() = 2.1;
```

Comment:
Changing the implementation did not require the test to change :)

The change to the test is also stylistic and for consistency with `determinant_and_inverse`: I am switching from a "column major" style ordering where the first index varies fastest, to a "row major" style ordering where the last index varies fastest. It's true that I swapped indices rather than reordering lines of code, and therefore the matrix being passed into the `determinant` function got transposed. However, as expected, its determinant doesn't change (within roundoff). This change to the test is independent from the change to the implementation.

If it gives you increased confidence, I can change this commit to reorder lines of code instead of swapping indices: this way the matrix will be identical, and only the way it is displayed will change.